### PR TITLE
Use dotenv for selecting models, improve log formatting

### PR DESCRIPTION
**Pull request by bparees**
- [ ] Refactor
- [x] New feature
- [ ] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Allow use of .env files to specify which models to use.

Also fixes up stdout logging and cleans up some typos

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: ols.py [hunk 1]
```diff
@@ -20,8 +20,7 @@
 
 load_dotenv()
 
-instruct_model = os.getenv("INSTRUCT_MODEL", "ibm/granite-13b-instruct-v1")
-rag_model = os.getenv("RAG_MODEL", "ibm/granite-13b-chat-v1")
+base_completion_model = os.getenv("BASE_COMPLETION_MODEL", "ibm/granite-20b-instruct-v1")
```
Comment (vishnuchalla) [inline ols.py:23]: Why do we want to make it a global variable when it is only used one function?

**Comment by vishnuchalla** [comment]
overall changes look go to me. lgtm

**Review by vishnuchalla — APPROVED** [review]

File: ols.py (hunk 1 above)
Comment (bparees) [reply ols.py:23]: long term we almost certainly don't(i doubt that function sticks around either).  but for now i was trying to minimize the changes to the overall code itself while adding in the configurability bits.

**Pull request merged**